romana-install/roles/romana/install-master/templates/config/romana.conf.yml

```diff
@@ -52,3 +52,36 @@ services:
     config:
       lease_file : "/etc/ethers"
       wait_for_iface_try: 6
+  - service: policy
+    api:
+      host: localhost
+      port: 9605
+    config:
+      store: 
+        type: mysql
+        host: 127.0.0.1
+        username: root
+        password: {{ stack_password }}
+        database: topology
+  - service: kubernetesListener 
```

Comment:
Does this replace the k8s-listener on master that would listen to namespace changes?